Mathlib/Data/Nat/Defs.lean

```diff
@@ -343,7 +330,7 @@ lemma add_succ_lt_add (hab : a < b) (hcd : c < d) : a + c + 1 < b + d := by
 #align nat.add_succ_lt_add Nat.add_succ_lt_add
 
 theorem le_or_le_of_add_eq_add_pred (h : a + c = b + d - 1) : b ≤ a ∨ d ≤ c := by
-  rcases le_or_lt b a with h' | h' <;> [left; right]
+  rcases (Nat.lt_or_ge a b).symm with h' | h' <;> [left; right]
```

Comment:
Can you add that as a file? Better let users not play the game of "Does this lemma have a `Nat` version or should I be reproving it?"